src/graph_notebook/neptune/client.py

```diff
@@ -185,13 +208,10 @@ def sparql_cancel(self, query_id: str, silent: bool = False):
     def get_gremlin_connection(self, transport_kwargs) -> client.Client:
         nest_asyncio.apply()
 
-        uri = f'{self._http_protocol}://{self.host}:{self.port}/gremlin'
-        request = self._prepare_request('GET', uri)
-
-        ws_url = f'{self._ws_protocol}://{self.host}:{self.port}/gremlin'
-
-        traversal_source = 'g' if "neptune.amazonaws.com" in self.host else self.gremlin_traversal_source
-        return client.Client(ws_url, traversal_source, headers=dict(request.headers), **transport_kwargs)
```

Comment:
Please replace the `**transport_kwargs` parameter in the GremlinPython client call here. We use transport args for certain Gremlin query options such as `--max-content-length`.